library/log_validation.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Need a proper gpl header. Here's an example (basically place you name/info in place of mine/rod's):
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#Page-Level_DocBlock